### DISCUSSION TITLE
There are parts where grouped elements are cutted off.

### DISCUSSION THREAD

**Discussion by rkawkxkdwlq** [fabricjs/fabric.js#8860] [Q&A]
![image](https://user-images.githubusercontent.com/24672968/234151748-d12f90d1-931d-4d77-9b49-b86f0416e261.png)

Some parts of the grouped elements are truncated as follows.
The height of the truncated arrow element only applies to the middle torso, so the height of the group is small, so it seems to be truncated.
How do I make it work the same as it is now, but not cut off?

**Comment by ShaMan123** [comment]
make sure you have `exactBoundingBox` set to true.
Might be a group thing so try v6
